### PR TITLE
Text Changes: Replace Current theme with Active theme in user facing strings

### PR DESCRIPTION
**Pull request by audrasjb**
Trac ticket: https://core.trac.wordpress.org/ticket/54770

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/themes.php [hunk 1]
```diff
@@ -133,8 +133,8 @@
 		'<p>' . __( 'From this screen you can:' ) . '</p>' .
 		'<ul><li>' . __( 'Hover or tap to see Activate and Live Preview buttons' ) . '</li>' .
 		'<li>' . __( 'Click on the theme to see the theme name, version, author, description, tags, and the Delete link' ) . '</li>' .
-		'<li>' . __( 'Click Customize for the current theme or Live Preview for any other theme to see a live preview' ) . '</li></ul>' .
-		'<p>' . __( 'The current theme is displayed highlighted as the first theme.' ) . '</p>' .
+		'<li>' . __( 'Click Customize for the currently active theme or Live Preview for any other theme to see a live preview' ) . '</li></ul>' .
```
Comment (costdev) [inline src/wp-admin/themes.php]: You knew this was coming 😂

What made you make this one different to the others?

**Review by costdev — APPROVED** [review]

File: src/wp-admin/themes.php (hunk 1 above)
Comment (audrasjb) [reply src/wp-admin/themes.php]: yeah, better to stick with the same wording :)

File: src/wp-admin/includes/class-wp-site-health.php [hunk 2]
```diff
@@ -653,7 +653,7 @@ public function get_test_theme_version() {
 						),
 						sprintf(
 							/* translators: 1: The default theme for WordPress. 2: The currently active theme. 3: The active theme's parent theme. */
-							__( 'To enhance your site&#8217;s security, you should consider removing any themes you&#8217;re not using. You should keep %1$s, the default WordPress theme, %2$s, your current theme, and %3$s, its parent theme.' ),
+							__( 'To enhance your site&#8217;s security, you should consider removing any themes you&#8217;re not using. You should keep %1$s, the default WordPress theme, %2$s, your currently active theme, and %3$s, its parent theme.' ),
```
Comment (costdev) [inline src/wp-admin/includes/class-wp-site-health.php]:
```suggestion
							__( 'To enhance your site&#8217;s security, you should consider removing any themes you&#8217;re not using. You should keep %1$s, the default WordPress theme, %2$s, your active theme, and %3$s, its parent theme.' ),
```

**Review by costdev — CHANGES_REQUESTED** [review]
Think that's everything from me now :+1:

**Comment by audrasjb** [comment]
Committed in https://core.trac.wordpress.org/changeset/52580

**Pull request closed without merging**